### ISSUE TITLE
Keyboard shortcut to change theme

### ISSUE THREAD

**Issue by mortenpi** [JuliaDocs/Documenter.jl#2097]
Similar to #2027, but for swapping between the light and dark themes.

**Comment by Hetarth02** [comment]
Can you please provide some more details regarding this?

Like, how should it look like ideally or maybe a keyboard shortcut you have in mind?

**Comment by mortenpi** [comment]
I guess this wouldn't really have a UI. But the UX here would be, yeah, you press the shortcut, and the the changes to the other theme. It should probably take into account the user's system theme preference too.

I don't really have any opinion about the shortcut. Is there any prior-art anywhere? In VS Code `Ctrl+K` + `Ctrl+T` opens the theme selector, but that's not super-useful.

**Comment by Hetarth02** [comment]
Yeah, we would really need to come up with a unique keyboard shortcut.

**Comment by Hetarth02** [comment]
I will take a look into this.

**Comment by Hetarth02** [comment]
@mortenpi I took a look into this, and found out that most websites don't tend to implement this kind of feature.

This could be due to the fact that, shortcuts are used for something which is likely to repeated many times.

And it is true, user won't be changing themes much often. So, I suggest we should think some more before starting to implement this feature.

**Comment by mortenpi** [comment]
> And it is true, user won't be changing themes much often.

I think that's a fair point. So I agree this wouldn't be a very high priority thing to do.

I do personally find myself swapping the themes quite a bit with Documenter (also because we can't share this preference across domains). So it was more motivated from a personal need. As such, though, it would be quite fine if the shortcut would be something pretty non-obvious and complicated.

**Comment by Hetarth02** [comment]
> As such, though, it would be quite fine if the shortcut would be something pretty non-obvious and complicated.

Yes, from what I have observed. Any shortcut which includes a `T` in it would open a new tab.

**Comment by odow** [comment]
This seems like a low priority that adds complexity. I'd be in favor of closing it.
